docs/toc.yml

```diff
@@ -291,6 +291,8 @@
       displayName: uninstall, sideload
     - name: Debug with runtime logging
       href: testing/runtime-logging.md
+    - name: Microsoft Office Add-in Debugger Extension for Visual Studio Code
```

Comment:
Side-note, do we need the full title in the TOC? I think the entry could be shortened to "Debugger Extension for Visual Studio Code", since all the product context is in the TOC already.